hack/lib/catalogsource.bash

```diff
@@ -119,7 +119,7 @@ spec:
         - -c
         - |-
           podman login -u $pull_user -p $token image-registry.openshift-image-registry.svc:5000 && \
-          /bin/opm registry add -d index.db --container-tool=podman --mode=replaces -b quay.io/openshift-knative/serverless-bundle:1.7.2,quay.io/openshift-knative/serverless-bundle:1.8.0,quay.io/openshift-knative/serverless-bundle:1.9.0,quay.io/openshift-knative/serverless-bundle:1.10.0,quay.io/openshift-knative/serverless-bundle:1.10.1,quay.io/openshift-knative/serverless-bundle:1.11.0,registry.ci.openshift.org/openshift/openshift-serverless-v1.12.0:serverless-bundle,registry.ci.openshift.org/openshift/openshift-serverless-v1.13.0:serverless-bundle,quay.io/openshift-knative/serverless-bundle:1.14.0,registry.ci.openshift.org/openshift/openshift-serverless-v1.15.0:serverless-bundle,image-registry.openshift-image-registry.svc:5000/$OLM_NAMESPACE/serverless-bundle && \
+          /bin/opm registry add -d index.db --container-tool=podman --mode=replaces -b docker.io/markusthoemmes/serverless-bundle:1.15-stop,image-registry.openshift-image-registry.svc:5000/$OLM_NAMESPACE/serverless-bundle && \
```

Comment:
project yaml should have the previous and the current versions and we can use them here, correct?

Reply:
Good idea! Let's do that as a followup. I'll need to discuss this "stop" image stuff with others first, but if we land that it'd be great to get this "bump" automated!

Reply:
Should that be `1.16-stop` instead of `1.15-stop`? (Looking at the stopbundle.Dockerfile below). 
Otherwise this approach looks good to me. And +1 for @skonto 's suggestion.

Reply:
No, in this case you need 1.15 (the "old" version) and the current one (which is 1.16, which is the internally built one).

Reply:
Done.